dspace/config/spring/api/qaevents.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:context="http://www.springframework.org/schema/context"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans
+           http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
+           http://www.springframework.org/schema/context
+           http://www.springframework.org/schema/context/spring-context-2.5.xsd">
+
+    <context:annotation-config /> <!-- allows us to use spring annotations in beans -->
+
+    <bean id="qaEventsDao" class="org.dspace.qaevent.dao.impl.QAEventsDAOImpl" />
```

Comment:
I like pretty much how we define external sources. We can add a different external source xml file with beans without having to change the original one and voilá, I've new external source based on new settings. I would like to ask you, because I can't understand based on this file specifically if this works this way. If it does, doesn't since the integration is with openaire, do you think it makes sense to rename the file to be specific for openaire qaevents beans?

Reply:
Hi @paulo-graca , yes your suggestion makes sense. I would suggest to restructure this file in the way that you suggest (a qaevents.xml for the "general configuration of the feature", a openaire-qaevents.xml for what is specific of openaire, etc.) in the coming PRs where we will introduce other sources (COAR Notify, DSpace Users)